### PR TITLE
Color change of the title bar on mouse over 

### PR DESCRIPTION
**Pull request by IbrahimFradj**
Related to #240 

The title bar changes color when the cursor is on it.

I added a hover color in the css.

Here is the result.
color

### CODE REVIEW

File: packages/react-components/style/output-components-style.css [hunk 1]
```diff
@@ -10,6 +10,9 @@
     display: grid;
     grid-template-rows: 25px 1fr;
 }
+.widget-handle :hover {
+    background-color: var(--theia-selection-background);
```
Comment (ebugden) [inline packages/react-components/style/output-components-style.css:14]: Please change the colour to something very subtle (ex. slightly lighten the bar colour) like in the example in the issue description #240. Here's the information priority that should be respected in this case:
1. **What action is possible:** The user should first notice that the cursor changes to a cross with arrows (this tells them they can click to move the panel). The cursor change is the most important piece of information.
2. **Where the action is possible:** The reason the title bar changes colour is just to show the the user the full area where they can hover to move the panel. It is secondary information so the colour change should be subtle. Right now the bright blue means the user is unlikely to notice the cursor change.

**Review by ebugden — CHANGES_REQUESTED** [review]
Behaves as expected for me! The selection area doesn't look elegant, but that's because we still need #239 so in my opinion elegance isn't required for merging this PR. 

After the colour adjustment, I'm happy to have this merged! 🌻

File: packages/react-components/style/output-components-style.css (hunk 1 above)
Comment (IbrahimFradj) [reply packages/react-components/style/output-components-style.css:14]: Alright, I understand. Thank you for your comment.  changes are coming soon

Comment (ebugden) [reply packages/react-components/style/output-components-style.css]: Thanks for the update. I like the new colour you chose!

**Review by ebugden — CHANGES_REQUESTED** [review]
Before we merge, you need to squash (combine) the two commits in this PR into one commit. You can do this by interactively rebasing the commits using git.

New colour looks good! 🌈

**Review by bhufmann — APPROVED** [review]
Looks good to me. Thanks for the UX improvement.

**Review by PatrickTasse — APPROVED** [review]

**Pull request merged**